improver/standardise.py

```diff
@@ -236,6 +236,15 @@ def _discard_redundant_cell_methods(cube: Cube) -> None:
 
         cube.cell_methods = updated_cms
 
+    @staticmethod
+    def _remove_long_name_if_standard_name(cube: Cube) -> None:
+        """
+        Remove the long_name attribute from cubes if the cube also has a standard_name defined
+        """
+
+        if cube.standard_name and cube.long_name:
+            cube.long_name = None
+
```

Comment:
@gavinevans do we need to flag this change at a joint stand-up with the Bureau? We're changing a default behaviour here (assuming they use Standardise) rather than something optional.

Reply:
I've flagged this with the Bureau, and we've got the green light to merge this right now.